docs/content/preview/troubleshoot/nodes/disk-full.md

```diff
@@ -1,25 +1,85 @@
 
-Starting with v2.18.1, you can prevent repeated WAL file allocation in a crash loop by using the [--reuse_unclosed_segment_threshold_bytes](../../../reference/configuration/yb-tserver/#reuse-unclosed-segment-threshold-bytes) flag.
+YugabyteDB data drive doesn't rely on the local root file system for storing data. Instead, YugaByteDB data directories are typically mounted on separate file systems to ensure isolation and manageability.
 
-If a tablet's last WAL file size is less than or equal to this threshold value, the bootstrap process will reuse the last WAL file rather than create a new one. To set the flag so that the last WAL file is always reused, you can set the flag to the current maximum WAL file size (64MB), as follows:
+The flags `max_disk_throughput_mbps` and `reject_writes_min_disk_space_mb` determine the amount of disk space that is considered too low. The default value is 3GB.
 
-```sh
-reuse_unclosed_segment_threshold_bytes=67108864
+The YB-TServer logs contain the following message when the system nears the threshold (18GB):
+
+```output
+W0829 15:35:51.325239 1986375680 log.cc:2186] Low disk space on yb-data/tserver/wals/table-7457ebcd745e4ea89375a30406f2c188/tablet-749e4d78244c43d2bba9cdb8505b732f/wal-000000001. Free space: 3518698209 bytes
+```
+
+and the following message when there is no more space left:
+
+```output
+0829 15:35:51.325239 1986375680 log.cc:2181] Not enough disk space available on yb-data/tserver/wals/table-7457ebcd745e4ea89375a30406f2c188/tablet-749e4d78244c43d2bba9cdb8505b732f/wal-000000001. Free space: 3518698209 bytes
+```
+
+## Issue
+
+### Write to tablet rejected due to insufficient disk space
+
+In YugabyteDB {{<release "2024.1.0.0">}} and later, insert, update, or delete to tables fail due to insufficient disk space with the following error:
+
+```output
+IO error (yb/tserver/tablet_service.cc:2288): Write to tablet 749e4d78244c43d2bba9cdb8505b732f rejected. Node 9a24d9c8493b44caa97ba4ae06e5829d has insufficient disk space
 ```
+
+## Recommended recovery actions
+
+**Option 1**: Increase capacity
+
+YugabyteDB allows you to expand the cluster in an online manner. There are two ways to scale the cluster:
+
+- Scale up
+    You can scale up by switching to bigger machines, or by adding more storage disks. If you are using YugabyteDB Anywhere, follow the instructions in [expanding universe node disk capacity with YugabyteDB Anywhere](https://support.yugabyte.com/hc/en-us/articles/5463616207757-Expanding-universe-node-disk-capacity-with-YugabyteDB-Anywhere-platform)
+
+- Scale out
+    You can scale out by adding more nodes to the cluster. This will cause the YugabyteDB load balancer to rebalance the tablets and the data across more nodes.
```

Comment:
```suggestion

    You can scale out by adding more nodes to the cluster. This will cause the YugabyteDB load balancer to rebalance the tablets and the data across more nodes.
```